### PR TITLE
Add support for include_institution_data flag

### PR DESCRIPTION
**Pull request by notthefakestephen**
/institutions/search
/institutions/get_by_id
/institutions/get

### CODE REVIEW

File: plaid/api/institutions.py [hunk 1]
```diff
@@ -7,7 +7,7 @@ class Institutions(API):
     (`HTTP docs <https://plaid.com/docs/api/#institutions>`__)
     '''
 
-    def get(self, count, offset=0):
+    def get(self, count, offset=0, _options=None):
         '''
         Fetch all Plaid institutions, using /institutions/all.
```
Comment (mattiskan) [inline plaid/api/institutions.py]: It doesn't look like `_options` is used anywhere. Please confirm that this is on purpose.

File: tests/integration/test_assets.py [hunk 2]
```diff
@@ -59,6 +59,7 @@ def test_full_flow():
     report = response['report']
     assert report is not None
 
+    print (f"hi!" {report})
```
[the remark this answers is not in the document]
Comment (notthefakestephen) [reply tests/integration/test_assets.py]: Was trying to debug why the assets test is failing. Will remove after investigation

File: plaid/api/institutions.py (hunk 1 above)
Comment (notthefakestephen) [reply plaid/api/institutions.py]: it's used in this tests `test_get_by_id_with_include_institution_data`

Comment (mattiskan) [reply plaid/api/institutions.py]: Sorry, I mean it's not passed anywhere — should this not be part of the request?

Comment (notthefakestephen) [reply plaid/api/institutions.py]: oops will fix

File: tests/integration/test_assets.py [hunk 3]
```diff
@@ -102,6 +101,18 @@ def test_full_flow():
...
 
+def name_exists_for_some_some_transaction(report):
+    i = 0
+    while i < len(report['items'][0]['accounts']):
+        if len(report['items'][0]['accounts'][i]['transactions']) > 0:
+            return (report['items'][0]['accounts'][i]
+                    ['transactions'][0]['name']
+                    is not None)
+        i = i + 1
+
```
Comment (mattiskan) [inline tests/integration/test_assets.py]: Sorry for style-comment, can we please use a for-each loop here?

File: tests/integration/test_assets.py [hunk 4]
```diff
@@ -62,8 +62,7 @@ def test_full_flow():
     # The transactions in an Asset Report with Insights should have a non-null
     # `name` (when available).
     assert (
-        report['items'][0]['accounts'][0]['transactions'][0]['name']
-    ) is not None
+        name_exists_for_some_some_transaction(report)) is True
```
Comment (mattiskan) [inline tests/integration/test_assets.py]: `is True` is redundant, `assert name_exists_for_some_some_transaction(report)` does the same.

File: tests/integration/test_assets.py [hunk 5]
```diff
@@ -102,6 +101,18 @@ def test_full_flow():
     assert removed
 
 
+def name_exists_for_some_some_transaction(report):
```
Comment (mattiskan) [inline tests/integration/test_assets.py]: "some" is repeated twice in name

File: tests/integration/test_assets.py (hunk 3 above)
Comment (notthefakestephen) [reply tests/integration/test_assets.py]: ⚡️

File: tests/integration/test_assets.py (hunk 4 above)
Comment (notthefakestephen) [reply tests/integration/test_assets.py]: ⚡️

File: tests/integration/test_assets.py (hunk 5 above)
Comment (notthefakestephen) [reply tests/integration/test_assets.py]: ⚡️

**Pull request merged**